### DISCUSSION TITLE
Question on jupyter tutorials

### DISCUSSION THREAD

**Discussion by bbermond73** [QuSpin/QuSpin#732] [Q&A] [answered]
Hello everyone,
I recently discovered quspin and was willing to learn or personal research projects.
While downloading and learning through the tutorials jupyter noteook, I encountered the following message in several pre-coded ligns results:

/var/folders/cl/tczn7vb11td_5sh0sbbyz85r0000gn/T/ipykernel_10741/3628100162.py:19: UserWarning: Test for symmetries not implemented for <class 'quspin.basis.basis_1d.spin.spin_basis_1d'>, to turn off this warning set check_symm=False in hamiltonian
  H_1 = hamiltonian(static_terms, dynamic_terms, basis=basis_1,)

Is it something normal or is it a problem of installation?

thank you or your support

Cheers

**Comment by SS007-M** [comment, marked as answer]
This is a normal warning that test for symmetries not implemented. If you add ' check_symm=False' in the hamiltonian, this will not show anymore.
 H_1 = hamiltonian(static_terms, dynamic_terms, basis=basis_1,check_symm=False)
